Grocer/Grocer/AppDelegate.swift

```diff
@@ -14,9 +14,62 @@ class AppDelegate: UIResponder, UIApplicationDelegate {
 
     var window: UIWindow?
 
+    static var main: AppDelegate {
```

Comment:
Are the changes in AppDelegate relevant/necessary for the model?

Reply:
We used that to test. I will clean up

Reply:
Okay rad, once you remove that Ill approve. The rest of it looks awesome!